src/Microsoft.ML.FastTree/Training/EnsembleCompression/LassoBasedEnsembleCompressor.cs

```diff
@@ -15,7 +15,8 @@ namespace Microsoft.ML.Trainers.FastTree.Internal
     /// https://www-stat.stanford.edu/~hastie/Papers/glmnet.pdf
     /// </summary>
     /// <remarks>Author was Yasser Ganjisaffar during his internship.</remarks>
-    public class LassoBasedEnsembleCompressor : IEnsembleCompressor<short>
+    [BestFriend]
```

Comment:
>[BestFriend] [](start = 4, length = 12)

While this probably should be internal, not best friend probably?

Reply:
Yes, you're right.

---
In reply to: [251186572](https://github.com/dotnet/machinelearning/pull/2243#discussion_r251186572) [](ancestors = 251186572)